account_credit_control/models/mail_message.py

```diff
@@ -7,8 +7,8 @@
 class MailMessage(models.Model):
     _inherit = "mail.message"
 
-    def message_format(self):
-        result = super().message_format()
+    def message_format(self, format_reply=True):
+        result = super().message_format(format_reply)
```

Comment:
Respect keyword arguments

```suggestion
        result = super().message_format(format_reply=format_reply)
```

Reply:
@pedrobaeza , I've fixed the commit message and keyword arguments as you mentioned above. Please review it.